.github/workflows/linter.yml

```diff
@@ -21,4 +27,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.60.1
```

Comment:
Shall we add caching here also something like below?
```
      - uses: actions/checkout@v4
      - uses: actions/cache@v4
        with:
          path: |
            ~/.cache/go-build
            ~/go/pkg/mod
          key: ${{ runner.os }}-golint-${{ hashFiles('**/go.sum') }}
          restore-keys: |
            ${{ runner.os }}-golint-
      - name: golangci-lint
        run: |
              make lint
```

Reply:
DAMNNN Man! Never thought this too could be added . Yeah we can add cache for linting as well.
Confirm me and I will do the changes

Reply:
@lucifercr07 give me a go to implement this